src/AppBundle/UseCase/RegisterUser.php

```diff
@@ -44,7 +36,7 @@ public function execute(RegisterUserCommand $command): User
             $user->setImage($command->getImage());
         }
 
-        $this->userRepository->add($user);
```

Comment:
It's a repository, not a collection itself, so I would rather stay with "userRepository" variable.

Reply:
ok 👍